staging_service/autodetect/Mappings.py

```diff
@@ -91,10 +91,12 @@ def _add_gzip(extension_list):
     EXCEL: ["xls", "xlsx"],
     ZIP: ["zip", "tar", "tgz", "tar.gz", "7z", "gz", "gzip", "rar"],
     SBML: ["smbl"],
-    # Custom File Types?
-    MEDIA: ["tsv", "xls", "xlsx"],
-    PHENOTYPE: ["tsv"],
-    ESCHER: ["json"],
-    ANNOTATIONS: ["tsv"],
-    FBA: ["tsv", "xls", "xlsx", "smbl"],
 }
+
```

Comment:
What's the purpose of this ?

Reply:
I assume you mean the `extension_to_type_mapping` - here it's checking that each extension is mapped to exactly one type. In the next PR this will be used to add the file type to the returned extension mapping.